Improve list of upcoming events on homepage

ptheywood opened the pull request:
Upcoming events are listed in chronological order, rather than reverse chrono.
The Javascript which filters out past-events has been modified that it behaves as intended in Firefox.
The homepage list has been filtereted to not output the html for past-events (at the time of compilation).

Closes #53

ptheywood committed:
Improve list of upcoming events on homepage

Upcoming events are listed in chronological order, rather than reverse chrono.
The Javascript which filters out past-events has been modified that it behaves as intended in Firefox.
The homepage list has been filtereted to not output the html for past-events (at the time of compilation).

Closes #53

twinkarma commented:
@ptheywood  The date pre-filtering in the event-list-upcoming is ok. 

I'd suggest changing the date formatting from the source instead of fixing in JS. See #55

twinkarma commented:
@ptheywood Changing from the template rather than source, sorry

ptheywood commented:
That does make much more sense. Will make the change now.

ptheywood committed:
Output Time as YYYY-MM-DD from jekyll rather than limiting the JS.

This does not currently consider time (or timezone), and will show the event until the end of the date

We may wish to add start and end date based filtereing for multi-day events in the future?
